### PR TITLE
Added Portuguese (pt_PT) translation

### PR DESCRIPTION
**Pull request by rmbb**
Complete translation to Portuguese (pt_PT). Based on the incomplete work of Portuguese Brazilian (pt_BR).

### CODE REVIEW

File: lang/languages.xml [hunk 1]
```diff
@@ -31,7 +31,7 @@ List of available languages for your wallet's seed:
         <language display_name="日本語" locale="ja_JP" wallet_language="日本語" flag="/lang/flags/japan.png" qs="none"/>
         <language display_name="Nederlands" locale="nl_NL" wallet_language="Nederlands" flag="/lang/flags/netherlands.png" qs="none"/>
         <!-- <language display_name="Polski" locale="pl_PL" wallet_language="English" flag="/lang/flags/poland.png" qs="none"/> -->
-        <language display_name="Português (BR)" locale="pt-br_BR" wallet_language="Português" flag="/lang/flags/brazil.png" qs="none"/>
+        <language display_name="Português (BR)" locale="pt-br_BR" wallet_language="Português (BR)" flag="/lang/flags/brazil.png" qs="none"/>
```
Comment (erciccione) [inline lang/languages.xml]: Please leave this string as it is or it won't be correctly processed

File: lang/languages.xml [hunk 2]
```diff
@@ -43,4 +43,5 @@ List of available languages for your wallet's seed:
         <language display_name="Slovensky" locale="sk_SK" wallet_language="English" flag="/lang/flags/slovakia.png" qs="none"/>
         <language display_name="Arabic" locale="ar_AR" wallet_language="English" flag="/lang/flags/egypt.png" qs="none"/>
         <language display_name="Slovenski" locale="sl_SI" wallet_language="English" flag="/lang/flags/slovenia.png" qs="none"/>
+		<language display_name="Português (PT)" locale="pt-pt_PT" wallet_language="Português (PT)" flag="/lang/flags/portugal.png" qs="none"/>
```
Comment (erciccione) [inline lang/languages.xml]: on wallet_language use "Português" and align the strings with the above ones

File: lang/languages.xml (hunk 1 above)
Comment (rmbb) [reply lang/languages.xml]: Ok!

File: lang/languages.xml (hunk 2 above)
Comment (rmbb) [reply lang/languages.xml]: In my notepad++ looks ok, will see what went wrong.

File: monero-wallet-gui.pro [hunk 3]
```diff
@@ -332,6 +332,7 @@ TRANSLATIONS =  \ # English is default language, no explicit translation file
                 $$PWD/translations/monero-core_sk.ts \ # Slovak
                 $$PWD/translations/monero-core_ar.ts \ # Arabic
                 $$PWD/translations/monero-core_sl.ts \ # Slovenian
+				$$PWD/translations/monero-core_pt-pt.ts \ # Portuguese (Portugal)
```
[the remark this answers is not in the document]
Comment (rmbb) [reply monero-wallet-gui.pro]: In my notepad++ looks ok, will see what went wrong.

File: qml.qrc [hunk 4]
```diff
@@ -118,6 +118,7 @@
         <file>lang/flags/netherlands.png</file>
         <file>lang/flags/palestine.png</file>
         <file>lang/flags/poland.png</file>
+		<file>lang/flags/portugal.png</file>
```
[the remark this answers is not in the document]
Comment (rmbb) [reply qml.qrc]: In my notepad++ looks ok, will see what went wrong.

**Pull request closed without merging**